db/seeds/all/faq.yaml

```diff
@@ -25,6 +80,18 @@ Syntax:
       scoped per Object. This means any method called on the same object will see the same instance
       variable.
 
+  - - How can I pass a method as an argument to another method?  
+    - |
+      You can get a callable object from a method by calling <code>#method</code>
+      on the object, for example <code>"hello".method(:length)</code>. You obtain
+      a <code>method</code> object which behaves like a <code>Proc</code> which you
+      can pass to other functions. 
+
+  - - Why does `"b" == "a" || "b"` not return true?  
+    - |
+      The <code>||</code> operator returns the argument which equals to true. In this
```

Comment:
Well, is it? `nil` and `false` are the only objects that are considered falsey in Ruby, everything else is truthy. Besides that we only need to mention that this applies to conditions in control flow statements like `if` and in filtering methods such as `Enumerable#select`.